src/openapi/streaming/connection/transport/signalr-core-transport.js

```diff
@@ -4,7 +4,7 @@ import * as constants from '../constants';
 
 const LOG_AREA = 'SignalrCoreTransport';
 const NOOP = () => {};
-const RECONNECT_DELAYS = [0, 2000, 3000, 5000, 10000, 20000];
+const RECONNECT_DELAYS = [0, 2000, 3000, 5000, 10000, null];

```

Comment:
null means stop trying? what happens if null isn't present? I think this deserves a comment

Reply:
sure.. will add comment